src/Silex/WebTestCase.php

```diff
@@ -42,14 +41,14 @@ public function setUp()
     /**
      * Creates the application.
      *
-     * @return HttpKernel
+     * @return Application
```

Comment:
ah thanks, so it should be `HttpKernelInterface`? (since Application does not extend `HttpKernel`)